### DISCUSSION TITLE
When to use outdoor or indoor sensors?

### DISCUSSION THREAD

**Discussion by PovilasID** [dolezsa/thermal_comfort#324] [Q&A]
Should I use indoor with indoor or outdoor with outdoor or mix to get an accurate dewpoint?
I am trying to prevent condensation water damage in place that I do not stay in. 
[Mold indictor](https://www.home-assistant.io/integrations/mold_indicator/) instructs to use indoor temperature and humidity and outdoor temperature. 
But in the [configuration documentation](https://github.com/dolezsa/thermal_comfort/blob/2.1/documentation/config_flow.md) you list both outdoor temperature and humidity... which got me confused...  Why mold sensor is using 3 sensors and you only two calculate dewpoint and if you are not automatically pulling in outdoor temp and humidity from the forecast (outdoor sensors in the docs)?

**Comment by housemaister** [comment]
Hi, I just came across your question while configuring thermal_comfort as well as mold_indicator.
I guess you are confusing the two. They have different purpose. Both are calculating different additional attributes and therefore need different input attributes.
Mold uses indoor temperature and humidity as well as outdoor temperature.
Thermal comfort uses temperature and humidity of one room (or outdoor).

**Reply by PovilasID** [reply]
Mold indicator has two attributes one called 'Dewpoint' % and other 'Estimated critical temp' deg C. Thermal comfort has a parameter `sensor.thermal_comfort_dew_point`  deg C. 
Is there rule of thumb that I could turn into an automation to turn up the heat or open a window and if so on which one of those I should trigger and what inputs should the one to use should have to have effective mold prevention?

**Reply by housemaister** [reply]
The dew point is calculated from room temperature and humidity; and should be basically the same for thermal_comfort as well as mold_indicator. If your room temperature gets too low near the dew point or there is a cold wall with temperatures near or below the dew point you have the risk for mold there and you should turn up heating.
For opening a window the comparison of absolute humidity given by thermal comfort from your room to the same of outside might help. You should get drop of rooms humidity if the absolute humidity is significant below the rooms one.
